python/tvm/contrib/cc.py

```diff
@@ -231,6 +234,21 @@ def _linux_compile(output, objects, options, compile_cmd, compile_shared=False):
     else:
         if compile_shared or output.endswith(".so") or output.endswith(".dylib"):
             cmd += ["--shared"]
+
+        compute_version = "".join(
+            get_target_compute_version(Target.current(allow_none=True)).split(".")
+        )
+        cmd += ["-gencode", f"arch=compute_{compute_version},code=sm_{compute_version}"]
+        cmd += ["-O3"]
+        cmd += ["-std=c++17"]
+        cmd += ["-Xcompiler=-fPIC"]
+        cmd += ["-Xcompiler=-fno-strict-aliasing"]
+        cuda_ver = get_cuda_version()
+        if cuda_ver >= (11, 2):
+            cmd += ["-t " + str(multiprocessing.cpu_count())]
+        cmd += ["-DCUTLASS_ENABLE_TENSOR_CORE_MMA=1"]
```

Comment:
these should be passed through `options`

Reply:
Sure. Will do.